docs/.vuepress/config.js

```diff
@@ -6,27 +6,46 @@ module.exports = (_ctx) => ({
     '/': {
       lang: 'en-US',
       title: 'MetaMask Docs',
-      description: 'Developer documentation for the MetaMask Ethereum wallet'
-    }
+      description: 'Developer documentation for the MetaMask Ethereum wallet',
```

Comment:
This file was updated to display the new pages in the sidebar, and while I was here I linted it with `prettier` and removed a useless function.

Reply:
Thank you!